test/jit/test_backends.py

```diff
@@ -465,3 +465,126 @@ def test_save_load(self):
     @skipIfRocm
     def test_errors(self):
         self.selective_lowering_test.test_errors()
+
+
+# Unit Tests for backend with compiler
+class BasicModuleAdd(torch.nn.Module):
+    """
+    A simple add Module used to test to_backend lowering machinery.
+    """
+
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, h):
+        return x + h
+
+# This is ignored in IS_WINDOWS or IS_MACOS cases. Hence we need the one in TestBackends.
+@unittest.skipIf(TEST_WITH_ROCM or IS_SANDCASTLE or IS_WINDOWS or IS_MACOS or IS_FBCODE,
+                 "Non-portable load_library call used in test")
+class JitBackendTestCaseWithCompiler(JitTestCase):
+    """
+    A common base class for JIT backend tests with compilers that contains common utility
+    functions for output comparison and serialization/deserialization.
+    """
+
+    def setUp(self):
+        super().setUp()
+        torch_root = Path(__file__).resolve().parent.parent.parent
+        p = torch_root / 'build' / 'lib' / 'libbackend_with_compiler.so'
+        torch.ops.load_library(str(p))
+        # torch.ops.load_library("//caffe2/test/cpp/jit:test_backend_compiler")
+        # Subclasses are expected to set up three variables in their setUp methods:
+        # module - a regular, Python version of the module being tested
+        # scripted_module - a scripted version of module
+        # lowered_modle - a version of module lowered to a backend
```

Comment:
Remove the commented lines?

Reply:
Including the bottom 4? I just copied those from the existing unit test.

Reply:
Good point. The bottom 4 should be kept.